clients/rust/tests/setup/stake.rs

```diff
@@ -2,7 +2,70 @@ use paladin_stake_program_client::{
     accounts::Stake, instructions::InitializeStakeBuilder, pdas::find_stake_pda,
 };
 use solana_program_test::ProgramTestContext;
-use solana_sdk::{pubkey::Pubkey, signer::Signer, system_instruction, transaction::Transaction};
+use solana_sdk::{
+    pubkey::Pubkey, signature::Keypair, signer::Signer, system_instruction,
+    transaction::Transaction,
+};
+
+use super::vote::create_vote_account;
+
+pub struct StakeManager {
+    // Stake account.
+    pub stake: Pubkey,
+    // Stake authority.
+    pub authority: Keypair,
+    // Validator account.
+    pub validator: Pubkey,
+    // Validator vote account.
+    pub vote: Pubkey,
+}
+
+impl StakeManager {
+    pub async fn new(context: &mut ProgramTestContext, config: &Pubkey) -> Self {
+        let mut manager = Self {
+            stake: Pubkey::default(),
+            authority: Keypair::new(),
+            validator: Pubkey::new_unique(),
+            vote: Pubkey::default(),
+        };
```

Comment:
nit: why not just initialize variables as you create the keys, rather than generating these ones that you're going to immediately discard?

```rust
let vote = create_vote_account(/* ... */);
let stake = create_vote_account(/* ... */);

Self {
    stake,
    vote,
}
```

Reply:
Good call!